src/mongoose_user_stats.erl

```diff
@@ -2,11 +2,36 @@
 
 -author('aleksander.lisiecki@erlang-solutions.com').
 
--export([report/0]).
+-behaviour(mongoose_service).
+
+-export([start/1, stop/0]).
+
+-define(TAB_NAME, persistent_user_info).
+
+% dummy is needed as it is not possible to create a mnesia table from single filed record.
+-record(?TAB_NAME, {client_id, dummy}).
```

Comment:
Well, we can just stick to `{key, value}` :)
Because over time we might want to store more user stats info there.